package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
```

Comment:
can you help me i don't know why not revert the changes

Reply:
sure, what I would do is run the command `git rebase master -i` this will give you a list of all your commits. Then go through and mark all the commits you do not want with a `d` this will delete them. Make sure you don't change the good ones

Reply:
oh and when you are done you will need to push back to this branch with the `-f` flag since you have changed the commits

Reply:
i not have any list of commit. The list is empty.